frontend/src/app/core/state/days/day.query.ts

```diff
@@ -0,0 +1,5 @@
+import { QueryEntity } from '@datorama/akita';
+import { DayState } from 'core-app/core/state/days/day.store';
+
+export class DayQuery extends QueryEntity<DayState> {
```

Comment:
I thought we don't want to follow this pattern of publicly available query objects (see #10760 ). Or did I misunderstood something here?

Reply:
Yes, correct. This pattern was introduced afterwards. I'll adapt the PR appropriately